zerver/migrations/0326_backfill_realmauditlog_extradata_to_json_field.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2 on 2021-05-07 14:43
+import orjson
+import yaml
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_extra_data(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    RealmAuditLog = apps.get_model("zerver", "RealmAuditLog")
+
+    for data in RealmAuditLog.objects.filter(extra_data__isnull=False):
+        try:
+            extra_data_json = orjson.loads(data.extra_data)
+        except orjson.JSONDecodeError:
+            extra_data_json = yaml.safe_load(data.extra_data)
```

Comment:
What's the story with falling back to YAML here?  This definitely deserves a comment, since I would expect we'd only need that if the old code was bugged in some way.